src/ucp/proto/proto_init.c

```diff
@@ -315,7 +315,7 @@ void ucp_proto_init_memreg_time(const ucp_proto_common_init_params_t *params,
     if (context->rcache != NULL) {
         perf_node = ucp_proto_perf_node_new_data("rcache lookup", "");
 
-        *memreg_time = UCP_RCACHE_LOOKUP_FUNC;
```

Comment:
The only another usage of `UCP_RCACHE_LOOKUP_FUNC` is inside `ucp_wireup_mem_reg_cost`, shouldn't we use the value from the config there too? If so, then `UCP_RCACHE_LOOKUP_FUNC` can be removed.

Reply:
I'm afraid that it will break wire compatibility.
@brminich fixed similar issue here: https://github.com/openucx/ucx/pull/9273.

Reply:
But shouldn't it be same values? Maybe it worth applying the new value by wire compatible way (with peer version check)?

Reply:
we have it defined in wire-compatible way in wireup (`ucp_wireup_mem_reg_cost`). here wire compatibility is not important.